src/java/org/jivesoftware/openfire/plugin/session/RemoteClientSession.java

```diff
@@ -40,13 +40,16 @@
  *
  * @author Gaston Dombiak
  */
-public class RemoteClientSession extends RemoteSession implements ClientSession {
 
-    private long initialized = -1;
+/*
```

Comment:
Can you remove this comment - the git repo is the historical record of truth, rather than code comments.